tools/spelling/spelling_dictionary.txt

```diff
@@ -121,6 +121,7 @@ GCP
 GETting
 GLB
 GOAWAY
+Glacier
```

Comment:
'Glacier' shouldn't need to be in this file. You had to add it probably because it was misspelled earlier?

Reply:
It was.